### PR TITLE
Jwineman/react15

### PR DESCRIPTION
**Pull request by jwineman**
```
$ rm -rf node_modules
$ npm install
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,10 +67,8 @@
     "mocha": "^2.4.5",
     "normalizr": "^2.0.0",
     "pump": "^1.0.1",
-    "react": "^0.14.2",
-    "react-addons-css-transition-group": "^0.14.6",
```
Comment (jwineman) [inline package.json]: This is never used.

File: src/js/components/C3Wrapper/C3Wrapper.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
```
Comment (jwineman) [inline src/js/components/C3Wrapper/C3Wrapper.js]: Copied from an internal stash repo I can show you if you want.

File: src/js/containers/AnalyticsPage/AnaltyicsPage.js [hunk 3]
```diff
@@ -162,6 +162,7 @@ class AnaltyicsPage extends Component {
         <div>
           <Heading size={1}><FormattedMessage id="container.analyticsPage.title"/></Heading>
           <Tabs
+            active={REQUESTS_TAB}
```
Comment (jwineman) [inline src/js/containers/AnalyticsPage/AnaltyicsPage.js]: `active` is a required parameter in the new `cf-component-tab`

File: src/js/containers/BrowserCacheTTLCard/BrowserCacheTTLCard.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ class BrowserCacheTTLCard extends Component {
 
     handleChange(value) {
         let { dispatch } = this.props;
-        dispatch(asyncZoneUpdateSetting(SETTING_NAME, this.props.activeZoneId, value));
+        dispatch(asyncZoneUpdateSetting(SETTING_NAME, this.props.activeZoneId, value.value));
```
Comment (jwineman) [inline src/js/containers/BrowserCacheTTLCard/BrowserCacheTTLCard.js]: New cf-component-select passes the entire JSON blog, not just the value.

File: src/js/containers/MinifyCard/MinifyCard.js [hunk 5]
```diff
@@ -37,7 +37,7 @@ class MinifyCard extends Component {
         };
 
         checkboxValueList.forEach(function(value){
-            apiValueList[value] = 'on';
+            apiValueList[value.value] = 'on';
```
Comment (jwineman) [inline src/js/containers/MinifyCard/MinifyCard.js]: Same for cf-component-checkbox.

File: src/js/components/AnalyticCard/AnalyticCard.js [hunk 6]
```diff
@@ -40,7 +40,7 @@ class AnalyticCard extends Component {
               <CardSection>
                 <CardContent title={ title }>
                   <Text size="small" type="muted">{ description }</Text>
-                  <hr style={{ margin: '1rem 0' }} align="left" width="100%"/>
+                  <hr style={{ margin: '1rem 0' }} width="100%"/>
```
Comment (jwineman) [inline src/js/components/AnalyticCard/AnalyticCard.js]: `align` isn't a valid attribute for `<hr>`

File: src/js/containers/MinifyCard/MinifyCard.js (hunk 5 above)
Comment (thellimist) [inline src/js/containers/MinifyCard/MinifyCard.js]: let's change the name from `value.value` to `item.value` or `checkboxitem.value`

File: src/js/containers/BrowserCacheTTLCard/BrowserCacheTTLCard.js (hunk 4 above)
Comment (thellimist) [inline src/js/containers/BrowserCacheTTLCard/BrowserCacheTTLCard.js]: same here let's change the variable name for everything in 4505811f183f721e48da108a0d0e5d5dcf7ea186

File: src/js/components/C3Wrapper/C3Wrapper.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  componentWillReceiveProps(props) {
+    this.updateC3(props);
+  }
+
+  componentWillUnmount() {
+    this.chart.destroy();
+  }
+
+  render() {
+    return <div ref={c => this._container = c}/>;
```
Comment (thellimist) [inline src/js/components/C3Wrapper/C3Wrapper.js]: `{c => this._container = c}` this is so weird syntax. Let's make this clearer

**Pull request merged**